config.xml

```diff
@@ -61,6 +61,8 @@
                 <data android:scheme="https" android:host="s3.amazonaws.com" android:path="/outline-vpn/invite.html" />
             </intent-filter>
         </custom-config-file>
+
+        <hook type="after_platform_add" src="scripts/update_android_target_version.js" />
```

Comment:
It doesn't make a huge difference I guess but it looks like hooks can be shell scripts, in which case you could avoid the `exports` business in the script. Up to you.

Reply:
Good suggestion, done in 006d5ad.